app/views/shelters/show.html.erb

```diff
@@ -11,9 +11,12 @@
 
 <%= link_to "New Review", "/shelters/#{@shelter.id}/reviews/new", method: "get" %>
 <% @reviews.each do |review| %>
-  <h3><%= review.title %></h3>
-  <p><%= review.rating %></p>
-  <p><%= review.content %></p>
-  <p><%= review.picture %></p>
-  <p><%= review.user.name %></p>
+  <section id="review-<%= review.id %>">
+    <h3><%= review.title %></h3>
+    <p><%= review.rating %></p>
+    <p><%= review.content %></p>
+    <p><%= review.picture %></p>
+    <p><%= review.user.name %></p>
+    <%= link_to "Edit Review", "/shelters/#{@shelter.id}/#{review.id}/edit", method: "get", id: "edit-#{review.id}-link"%>
```

Comment:
Love this! Nice job figuring out how to id a link (since each edit link will technically be separate for each review)